### DISCUSSION TITLE
make cookie policies relaxed during dev mode in sveltekit-auth-helper

### DISCUSSION THREAD

**Discussion by thedelanyo** [supabase/supabase#15808] [Feature Requests]
I don't know but during development the `sb-whateverId-auth-token` cookie refused to be set when localhost is running with http, hence session is null on both server and client. This issue is halve solved when I changed from http to https on localhost. But then I noticed that session is Session on server but null on client, even with https, even when I implemented the `onAuthStateChange` in `onMount`.

Usually when setting and getting cookies, we make it relaxed on dev but secured and `http-only` during `!dev`.

**Comment by j4w8n** [comment]
Can you provide a minimum repo with this issue? I'm having no problems with http://localhost and cookies
